<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>402</width>
    <height>260</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>402</width>
    <height>260</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>FileSplit</string>
  </property>
  <property name="windowIcon">
   <iconset resource="filesplit.qrc">
    <normaloff>:/img/divide.png</normaloff>:/img/divide.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>9</y>
      <width>401</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>Split File Select</string>
    </property>
    <widget class="QLineEdit" name="lineEdit_SplitFileSelect">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>341</width>
       <height>20</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_SplitFileSelect">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>30</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="filesplit.qrc">
       <normaloff>:/img/folder.png</normaloff>:/img/folder.png</iconset>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>180</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Split Number</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_SplitNumber">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>180</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Change">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>210</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="filesplit.qrc">
      <normaloff>:/img/change.png</normaloff>:/img/change.png</iconset>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>90</y>
      <width>401</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>Save Folder Select</string>
    </property>
    <widget class="QLineEdit" name="lineEdit_SaveFolderSelect">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>341</width>
       <height>20</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_SaveFolderSelect">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>30</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="filesplit.qrc">
       <normaloff>:/img/folder.png</normaloff>:/img/folder.png</iconset>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>180</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Create File</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_FileNumber">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>180</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>402</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="filesplit.qrc"/>
 </resources>
 <connections/>
</ui>
